src/components/myPage/blockAccount/bolckAccount.style.ts

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+import { COLORS } from '../../../styles/constants/colors';
```

Comment:
```suggestion
import { COLORS } from '@/styles/constants/colors';
```
로 불러오는 방식 변경하는건 어떤가요?